package.json

```diff
@@ -7,7 +7,8 @@
 	},
 	"main": "castWebApi.js",
 	"scripts": {
-      	"cast-web-api": "node ./castWebApi.js"
+				"cast-web-api": "node ./castWebApi.js",
+				"start": "node castWebApi.js"
```

Comment:
Can you maybe explain what this does? Does it allow for 'cast-web-api start' and 'cast-web-api' in the install directory and it will start the api?

Reply:
This allows you to type 'npm start' and the server will start up. Setting a default action to the start command is typical for node applications.